docroot/modules/custom/foia_webform/src/Plugin/WebformHandler/FoiaEmailWebformHandler.php

```diff
@@ -0,0 +1,134 @@
+<?php
+
+namespace Drupal\foia_webform\Plugin\WebformHandler;
+
+use Drupal\file\Entity\File;
+use Drupal\webform\Plugin\WebformHandler\EmailWebformHandler;
+use Drupal\webform\WebformSubmissionInterface;
+use Drupal\node\Entity\Node;
+
+/**
+ * Emails a webform submission.
+ *
+ * @Webformhandler(
+ *   id = "foia_email",
+ *   label = @Translation("FOIA email"),
+ *   category = @Translation("Notification"),
+ *   description = @Translation("Sends a webform submission to appropriate the agency."),
+ *   cardinality = \Drupal\webform\WebformHandlerInterface::CARDINALITY_UNLIMITED,
+ *   results = \Drupal\webform\WebformHandlerInterface::RESULTS_PROCESSED,
+ * )
+ */
+class FoiaEmailWebformHandler extends EmailWebformHandler {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function sendMessage(WebformSubmissionInterface $webform_submission, array $message) {
+    // Get form submissions.
+    $data = $webform_submission->getData();
+    $error_message = NULL;
+    $context = [];
+
+    // If there is a file attachment, get the file URL.
+    if (isset($data['attachments_supporting_documentation'])) {
+      $file = File::load($data['attachments_supporting_documentation']);
+      $file_url = file_create_url($file->getFileUri());
+      $data['attachments_supporting_documentation'] = $file_url;
```

Comment:
A couple of comments:
1) In #69 I modified this field to be able to accept multiple attachments, so we'll need to modify the code here to handle that.
2) Will this actually 'attach' the file to the e-mail? Or simply include a url to the file? I ask because we're planning on deleting submission data after it is received by the component (another ticket), so a link to the file wouldn't work once that's in place